src/app/containers/Ad/Amp/index.jsx

```diff
@@ -36,29 +63,54 @@ const ampAdPropsDesktop = ({ service }) => ({
   json: JSON.stringify(constructAdJsonData({ service })),
 });
 
+const AMP_ACCESS_DATA = (endpoint) => ({
+  authorization: endpoint,
+  noPingback: true,
+});
+
+const AMP_ACCESS_FETCH = (service) => {
```

Comment:
Great work on this @thekp Just want to make a suggestion for a future enhancement. Right now its only Amp Ads that needs access to this data, so fetching here is fine however in the future we should have this data fetch at the page level. this way any component can access this amp-access data, this would make the toggles useable for services that do not have ads.

Nothing to do in this PR, just a future improvement.

Reply:
Thanks for notes and looking Chris  💯